presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix scheduling issue in join spill to improve memory used by unspilled data.
+* Fix spilled query exceeding local memory limit due to incorrect memory size calculation when spilled data is read by SingleStreamSpiller.
+* Fix deadlock problem that results into join + limit getting stuck when spilling is enabled.
+* Improve RowExpressionFormatter output to be closer to valid SQL. Constant expressions are now output as ``TYPE 'value'`` (e.g. ``BIGINT'2'``) using TypeConstructor (with single quotes appearing in the string escaped with ``''``), with the exception of ``VARBINARY`` constants, which use the binary literal syntax (e.g. ``X'abc123'``). ``LIKE`` expressions are now output as valid SQL.
+* Add support for instance count in function :func:`array_position`.
+* Add support for expressions with multiple ``IS NULL`` on the same variable.
+* Add support for more kinds of correlated subqueries.
```

Comment:
Should we enumerate the types of subqueries?

Reply:
Updated to `* Add coercion support when correlated symbols are in non-direct column reference type.`